locales/languages/en.json

```
@@ -586,8 +586,21 @@
     "nft_autodetect_desc": "Displaying NFT media & data may expose your IP address to centralized servers. Third-party APIs (like OpenSea) are used to detect NFTs in your wallet. This exposes your account address with those services. Leave this disabled if you don't want the app to pull data from those services.",
     "show_hex_data": "Show Hex Data",
     "show_hex_data_desc": "Select this to show the hex data field on the send screen.",
-    "enable_eth_sign": "Toggle eth_sign requests",
-    "enable_eth_sign_desc": "Turn this on to let dapps request your signature using eth_sign requests. eth_sign is an open-ended signing method that lets you sign an arbitrary hash, making it a dangerous phishing risk. Only sign eth_sign requests if you can read what you are signing and trust the origin of the request.",
+    "toggleEthSignBannerDescription": "You’re at risk for phishing attacks. Protect yourself by turning off eth_sign.",
+    "enable_eth_sign": "Eth_sign requests",
+    "enable_eth_sign_desc": "If you enable this setting, you might get signature requests that aren’t readable. By signing a message you don't understand, you could be agreeing to give away your funds and NFTs.",
+    "toggleEthSignModalBannerText": "If you've been asked to turn this setting on,",
+    "toggleEthSignModalBannerBoldText": " you might be getting scammed.",
+    "toggleEthSignModalCheckBox": "I understand that I can lose all of my funds and NFTs if I enable eth_sign requests.",
+    "toggleEthSignModalDescription": "Allowing eth_sign requests can make you vulnerable to phishing attacks. Always review the URL and be careful when signing messages that contain code.",
+    "toggleEthSignModalFormError": "This text is incorrect. Try again.",
+    "toggleEthSignModalFormLabel": "Enter “I only sign what I understand” to continue",
+    "toggleEthSignModalFormValidation": "I only sign what I understand",
+    "toggleEthSignModalTitle": "Use at your own risk",
+    "toggleEthSignModalEnable": "Enable",
+    "toggleEthSignModalLearnMore": "Learn more.",
+    "toggleEthSignOff": "OFF (Recommended)",
+    "toggleEthSignOn": "ON (Not recommended)",
```

Can you add the following button labels please?
```suggestion
    "toggleEthSignOn": "ON (Not recommended)",
    "toggleEthSignContinueButton": "Continue",
    "toggleEthSignEnableButton": "Enable",
```